components/offscreen/offscreen.js

```diff
@@ -1,15 +1,20 @@
 import { css, html, LitElement } from 'lit';
 import { RtlMixin } from '../../mixins/rtl/rtl-mixin.js';
 
-export const offscreenStyles = css`
-	.d2l-offscreen {
+const offscreenStyleDeclarations = css`
+		direction: var(--d2l-document-direction, ${document.dir === 'rtl' ? css`rtl` : css`ltr`});
```

Comment:
This is a cool solution! Just so I make sure I have this correct:
- `direction` will use `--d2l-document-direction` and fall back to `document.dir` if that isn't set... which would only be true if typography isn't included. What happens if typography is included _after_ `offscreen.js`, does that still work and use the variable?
- `--d2l-document-direction` dynamically changes when the `dir` attribute on the `html` element changes. I'm assuming that means any changes MUST modify the attribute only, and can't set the property or `document.dir` directly.
- `direction` in this style block controls `inset-inline-start` but has no effect on `left` which is only used as a fallback anyway for browsers who don't support logical properties.